app-frontend/src/app/pages/projects/edit/aoi-parameters/aoi-parameters.controller.js

```diff
@@ -112,28 +136,18 @@ export default class AOIParametersController {
         });
     }
 
-    updateProjectAOIs(polygonCoordinates) {
+    updateProjectAOIs(multipolygon) {
         if (this.projectAois && this.projectAois.length === 1) {
             // update existing aoi
             let aoiToUpdate = this.projectAois[0];
-            if (polygonCoordinates.length) {
-                aoiToUpdate.area = {
-                    geom: {type: 'MultiPolygon', coordinates: polygonCoordinates},
-                    srid: 3857
-                };
-            } else {
-                this.$log.error('An AOI must be composed of at least one polygon');
-            }
+            aoiToUpdate.area = multipolygon;
             this.aoiService.updateAOI(aoiToUpdate).then(() => {
```

Comment:
Hey @alkamin , I have made the changes to address this in the latest commit.  Please let me know if it needs further changes.